packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -98,6 +98,54 @@ const drawerFooter = ({
     );
 };
 
+const mobileDrawerFooter = ({
+    onStopButtonClick,
+    onRunButtonClick,
+    is_stop_button_visible,
+    is_stop_button_disabled,
+}) => {
+    return (
+        <div className='controls__section'>
+            <div className='controls__buttons'>
+                {is_stop_button_visible ? (
+                    <Button
+                        className='controls__stop-button'
+                        is_disabled={is_stop_button_disabled}
+                        text={localize('Stop bot')}
```

Comment:
As per the design, the text should only be `Stop`.

```suggestion
                        text={localize('Stop')}
```